llvm/lib/Target/SystemZ/SystemZTargetTransformInfo.cpp

```diff
@@ -1284,17 +1286,43 @@ InstructionCost SystemZTTIImpl::getInterleavedMemoryOpCost(
   return NumVectorMemOps + NumPermutes;
 }
 
-static int getVectorIntrinsicInstrCost(Intrinsic::ID ID, Type *RetTy) {
+static int
+getVectorIntrinsicInstrCost(Intrinsic::ID ID, Type *RetTy,
+                            const SmallVectorImpl<Type *> &ParamTys) {
   if (RetTy->isVectorTy() && ID == Intrinsic::bswap)
     return getNumVectorRegs(RetTy); // VPERM
+
+  if (ID == Intrinsic::vector_reduce_add) {
+    // Retrieve number and size of elements for the vector op
```

Comment:
Comments should be prose with proper punctuation.  (see https://llvm.org/docs/CodingStandards.html#commenting)

Reply:
Hi, i've re-worded the comments and added closing punctuation. Please let me know if this addresses your concerns.

Reply:
Thanks!